packages/cli/src/index.ts

```diff
@@ -16,6 +16,21 @@ const packageJson = require('../package.json');
 
 import program = require('commander');
 
+const MINIMUM_NODE_VERSION = 12;
+const COLOR_RED = '\x1b[31m';
+const COLOR_YELLOW = '\x1b[33m';
+const FORMAT_UNDERLINE = '\x1b[4m';
+const FORMAT_RESET = '\x1b[0m';
+
+const [nodeMajorVersion] = process.versions.node.split('.');
+if (parseInt(nodeMajorVersion, 10) < MINIMUM_NODE_VERSION) {
+  console.error(`${FORMAT_UNDERLINE}${COLOR_RED}You have an outdated node version!`);
```

Comment:
wouldnt it be a tick cleaner then to do this at the end here?
```suggestion
  console.error(`${FORMAT_UNDERLINE}${COLOR_RED}You have an outdated node version!${FORMAT_RESET}`);
```

Reply:
not 2 bothered, just a thought

Reply:
yes, we can do that